plugins/pci-mei/fu-pci-mei-plugin.c

```diff
@@ -21,6 +21,39 @@ struct _FuPciMeiPlugin {
 
 G_DEFINE_TYPE(FuPciMeiPlugin, fu_pci_mei_plugin, FU_TYPE_PLUGIN)
 
+static void
+fu_pci_mei_plugin_to_string(FuPlugin *plugin, guint idt, GString *str)
+{
+	FuPciMeiPlugin *self = FU_PCI_MEI_PLUGIN(plugin);
+	if (self->pci_device) {
+		fwupd_codec_string_append(str,
+					  idt,
+					  "PciDevice",
+					  fu_device_get_id(self->pci_device));
+	}
+	for (guint i = 0; i < 6; i++) {
+		g_autofree gchar *title = g_strdup_printf("Hfsts%u", i);
+		fwupd_codec_string_append_hex(
+		    str,
+		    idt,
+		    title,
+		    fu_memread_uint32(self->hfsts_buf[i], G_LITTLE_ENDIAN));
+	}
+	if (self->family != FU_MEI_FAMILY_UNKNOWN) {
```

Comment:
May as well show "Unknown" in this case too.

Reply:
Ya, it's the important bit I guess. Fixed.